py3status/modules/volume_status.py

```diff
@@ -23,10 +23,12 @@
         (default None)
     device: Device to use. Defaults value is backend dependent
         (default None)
+    device_is_speaker: Should this control the speakers (True) or the microphone (False)?
+        (default True)
     format: Format of the output.
-        (default '♪: {percentage}%')
+        (default '[\?if=device_is_speaker ♪|😮]: {percentage}%')
```

Comment:
I'd still like a nicer name, but the code now looks good apart from this issue.

I don't like microphone/speaker as they are device specific so something more input/output or source/sink

`is_output`?